pkgs/servers/monitoring/prometheus/xmpp-alerts.nix

```
@@ -1,17 +1,37 @@
-{ lib, fetchFromGitHub, pythonPackages }:
+{ lib
+, fetchFromGitHub
+, python3Packages
+, prometheus-alertmanager
+}:
 
-pythonPackages.buildPythonApplication rec {
+with python3Packages;
+
+buildPythonApplication rec {
```

```suggestion
buildPythonApplication rec {
```
applying with to the entire scope is an anti pattern.